.nf-core.yml

```diff
@@ -0,0 +1,4 @@
+# Remove when not needed anymore!
```

Comment:
just to make sure, this is still needed?

Reply:
it is